arch-intel-centerton-pci-id.patch

Add the PCI device ID for the Intel Centerton Integrated Legacy Block
(ILB).

This device ID is used by a number of drivers, including GPIO and
Multifunction Devices.

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index d93f417..d35b1f4 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2484,6 +2484,7 @@
 #define PCI_DEVICE_ID_INTEL_MRST_SD2	0x084F
 #define PCI_DEVICE_ID_INTEL_I960	0x0960
 #define PCI_DEVICE_ID_INTEL_I960RM	0x0962
+#define PCI_DEVICE_ID_INTEL_CENTERTON_ILB	0x0c60
 #define PCI_DEVICE_ID_INTEL_8257X_SOL	0x1062
 #define PCI_DEVICE_ID_INTEL_82573E_SOL	0x1085
 #define PCI_DEVICE_ID_INTEL_82573L_SOL	0x108F
